tests/TestCase/Model/Table/SentencesListsTableTest.php

```diff
@@ -198,7 +198,33 @@ function testAddSentenceToList_failsBecauseUserNotAllowed() {
         $after = $this->SentencesList->SentencesSentencesLists->find()
             ->where(['sentence_id' => 12, 'sentences_list_id' => 1])
             ->count();
-        
+
+        $this->assertFalse($result);
+        $this->assertEquals($after, $before);
+    }
+
+    function testAddSentenceToList_otherUserFailsBecauseEditableByNoOne() {
+        $before = $this->SentencesList->SentencesSentencesLists->find()
+            ->where(['sentence_id' => 12, 'sentences_list_id' => 6])
+            ->count();
+        $result = $this->SentencesList->addSentenceToList(12, 6, 4);
```

Comment:
Could you make the tests a little bit more easy to understand by using variables instead of numbers? For example:
```php
$userId = 4;
$listId = 6;
$sentenceId = 12;
$this->SentencesList->addSentenceToList($sentenceId, $listId, $userId);
```

This also ensures that you’re not mixing up numbers, forcing the use of same numbers in the `where()` clause and in the `addSentenceToList()` call.

Reply:
Sure. I'll try to find some time today. What about the other tests? Do you want them rewritten or should we let them as they are?